eZ/Publish/Core/settings/repository.yml

```diff
@@ -163,3 +164,10 @@ services:
 
     ezpublish.search.background_indexer:
         class: eZ\Publish\Core\Search\Common\BackgroundIndexer\NullIndexer
+
+    ezpublish.repository.relation_processor:
+        class: eZ\Publish\Core\Repository\Helper\RelationProcessor
+        arguments:
+            - "@ezpublish.api.persistence_handler"
+        calls:
+            - ["setLogger", ["@?logger"]]
```

Comment:
Please use single quotes for new service definition (file is inconsistent anyway due to my reviews ;) )